embedding/embedlite/embedding.js

```diff
@@ -107,15 +107,19 @@ pref("media.prefer-gstreamer", true);
 pref("media.gstreamer.enable-blacklist", false);
 // Disable X backend on GTK
 pref("gfx.xrender.enabled", false);
+
+// AZPC overrides, see EmbedLiteViewChild.cpp
 pref("embedlite.azpc.handle.viewport", true);
-pref("embedlite.azpc.handle.singletap", true);
-pref("embedlite.azpc.handle.longtap", true);
+pref("embedlite.azpc.handle.singletap", false);
+pref("embedlite.azpc.handle.doubletap", true);
```

Comment:
We could consider dropping this one altogether as ```sHandleDefaultAZPC.doubleTap``` is not referenced (i.e. embedlite.azpc.handle.doubletap doesn't do anything.

As a counterpart do we need that ```embedlite.azpc.json.doubletap``` in reality either. We could have hittesting done always.

Reply:
Right, the former is a dead setting. I can remove it.

For `embedlite.azpc.json.doubletap`: setting this to false will bring the browser behaviour more in line with Android FF (which, surprisingly to me, always seems to override `dblclick` handlers with zoom), so maybe it's worth keeping for that reason.

But I'm not really certain what the broader semantics of these settings are intended for. Perhaps this is the wrong setting for that?

Reply:
Let's drop the dead one.